copilot-chat-request.el

```diff
@@ -122,7 +121,7 @@ Argument DATA is whatever PARSER function returns, or nil."
                ("editor-version" . "Neovim/0.10.0")
                ("editor-plugin-version" . "CopilotChat.nvim/2.0.0")
                ("user-agent" . "CopilotChat.nvim/2.0.0"))
-    :parser 'json-read
+    :parser (apply-partially 'json-parse-buffer :object-type 'alist :false-object :json-false :null-object nil)
```

Comment:
`json.c` does not use `:json-false`, but `:false` is assigned by default, so there is no need to force a custom setting.